content/03-reference/01-tools-and-interfaces/02-prisma-client/02-generating-prisma-client.mdx

```diff
@@ -55,28 +55,30 @@ const prisma = new PrismaClient()
 
 ## The `@prisma/client` npm module
```

Comment:
Is `npm module` common usage here? I would have expected it to be called "npm package". 

(also applies to all the other content of course.)